client/.eslintrc.json

```diff
@@ -13,6 +13,15 @@
   "commonjs": true,
   "es6": true
   },
+  "overrides": [
+    {
+        "files": ["*.ts", "*.tsx"],
+        "rules": {
+            "no-undef": "off",
+            "import/extensions": "off"
```

Comment:
These 2 rules are handled via typescript, sometimes the logic is different than eslint and they go out of sync, better to keep typescript in charge here